### PR TITLE
Upgrade Docusaurus to help fix Netlify Timeout issues.

### PR DESCRIPTION
**Pull request by csestito**
## Status
Ready for review

## Related Issues
Netlify timeout during build

## Description
Some of the Netlify builds have been timing out. Steven discovered that the recent Docusaurus version will handle some errors better than the current version. Updated the Docusaurus version and ported over the changes that were made to the components, as well as removed components that no longer needed swizzled. Also updated the logo during dark mode. 

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: src/theme/DocItem/index.js [hunk 1]
```diff
@@ -51,36 +53,40 @@ function Headings({ headings, isChild }) {
...
+      image: metaImage,
+      keywords,
       hide_title: hideTitle,
       hide_table_of_contents: hideTableOfContents
     }
   } = DocContent;
 
   const issueTitle = `Issue with "${title}" in ${source}`;
-  const issueUrl = `https://github.com/demisto/content-docs/issues/new?labels=documentation&template=developer-documentation-issue.md&title=${issueTitle}`;
+  const issueUrl = `https://github.com/PaloAltoNetworks/cortex.pan.dev/issues/new?labels=documentation&template=developer-documentation-issue.md&title=${issueTitle}`;
```
Comment (fvigo) [inline src/theme/DocItem/index.js:79]: I think this is not correct

Comment (csestito) [reply src/theme/DocItem/index.js:79]: Fixed this locally, haven’t pushed the new built yet. 

In terms of needing to change the templates every time we upgrade the answer is yes. As upgrades come through, it changes the components so we need to reswizzle the components and if a particular component is changed we have to incorporate the necessary changes into the swizzled component. Nothing changed on my end debug wise, will push a new update through in the morning and prolong the build.

**Comment by glicht** [comment]
Build is failing. Not sure if it is related to your changes or not

**Comment by csestito** [comment]
Trying to fix what appears to be a CDN issue with the light mode logo, then I'll merge this

**Pull request merged**